### PR TITLE
optimization of assertmsg sender

### LINKED ISSUE #46: refactor in HelperAssert.sol to remove duplicate code

### PR DESCRIPTION
**Pull request by antks03**
@rappie 

close #46

### CODE REVIEW

File: src/helpers/HelperAssert.sol [hunk 1]
```diff
@@ -22,6 +22,10 @@ abstract contract HelperAssert is HelperBase {
         }
     }
 
+    function createErrorMessage(string memory aStr, string memory bStr, string memory operator, string memory reason) private pure returns (bytes memory) {
```
Comment (rappie) [inline src/helpers/HelperAssert.sol:25]: let's move this one to the bottom because it is a helper function

Comment (rappie) [reply src/helpers/HelperAssert.sol:25]: make it `internal` instead of `private` so we can use it in child contracts as well (like the foundry tests)

Comment (rappie) [reply src/helpers/HelperAssert.sol:25]: i wonder if "error message" is the best name, perhaps we should call it `createAssertFailMessage`? what do you think? :)

**Review by rappie — CHANGES_REQUESTED** [review]

File: src/helpers/HelperAssert.sol (hunk 1 above)
Comment (antks03) [reply src/helpers/HelperAssert.sol:25]: Lets's do it!!

**Review by rappie — APPROVED** [review]

**Pull request merged**